.github/workflows/pr-build.yaml

```diff
@@ -26,6 +26,12 @@ jobs:
         with:
           fetch-depth: 0
 
+      # On the Mac build machines, libssl 1.1 is installed
+      # but not linked in the path expected by .NET. Fix it now.
+      - name: Link libssl on Mac OS
+        run: ln -sn /usr/local/opt/openssl/lib/libssl.1.1.dylib /usr/local/lib/libssl.1.1.dylib
+        if: matrix.os == 'macOS-latest'
```

Comment:
Is this change permanent, does it affect the build agents after the build is over?

Reply:
I'm pretty sure the build agent VMs are reset before every build. If not then the `ln` would fail in subsequent builds (see my other comment).